meta/FlexibleSUSY.m

```diff
@@ -2533,18 +2530,75 @@ corresponding tadpole is real or imaginary (only in models with CP
                CXXDiagrams`IndexDiagramFromGraph[diagrams[[i,j]], graphs[[i]]],
                   graphs[[i]]
                 ], 16] <> " * " <>
-                ToString @ AMuon`CXXEvaluatorForDiagramFromGraph[diagrams[[i,j]], graphs[[i]]] <>
+                ToString @ AMM`CXXEvaluatorForDiagramFromGraph[diagrams[[i,j]], graphs[[i]]] <>
                 "::value({" <> muonIndex <> "}, context, qedqcd);\n"
          ];
       ];
 
+      leptonPhysicalMass =
+         If[GetParticleFromDescription["Leptons"] =!= Null,
+"template <typename Lepton>
+double leptonPhysicalMass(const softsusy::QedQcd& qedqcd, int idx)
```

Comment:
I'd propose to rename this function to `lepton_pole_mass` for consistency with our C++ function naming policy. Furthermore I think "pole" is better than "physical" here, because we use the term "pole" in most places in FS.

Reply:
Done in 87ab68bf65f2b7047f0cd0fd08497fb23b2f508f.